2023-09-11 15:57:16,092 - ERROR - Error GENERAL 10.224.113.129
2023-09-11 15:57:16,092 - ERROR - list index out of range
2023-09-11 16:03:40,533 - ERROR - Error GENERAL 10.224.113.129
2023-09-11 16:03:40,533 - ERROR - list index out of range
2023-09-11 17:14:15,938 - ERROR - Error en el FW 10.224.21.1
2023-09-11 17:14:15,938 - ERROR - 1054 (42S22): Unknown column 'getway' in 'field list'
2023-09-11 17:14:15,939 - ERROR - Error GENERAL 10.224.21.1
2023-09-11 17:14:15,939 - ERROR - 1054 (42S22): Unknown column 'getway' in 'field list'
2023-09-13 15:08:15,350 - ERROR - Traceback (most recent call last):
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Camaras/main.py", line 153, in get_devices_data
    CISCO_DEVICE_IP_URL = os.getenv('URL_CISCO_ip').format(red=red, ip=cisco_device_ip_adrress)
AttributeError: 'NoneType' object has no attribute 'format'

2023-09-13 15:16:02,397 - ERROR - Traceback (most recent call last):
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Camaras/main.py", line 169, in get_devices_data
    cisco_device_reachability = cisco_device_id_response['queryResponse']['entity'][0]['devicesDTO']['reachability']
KeyError: 'queryResponse'

2023-09-13 15:18:20,535 - ERROR - Traceback (most recent call last):
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Camaras/main.py", line 170, in get_devices_data
    cisco_device_reachability = cisco_device_id_response['queryResponse']['entity'][0]['devicesDTO']['reachability']
KeyError: 'queryResponse'

2023-09-14 10:08:15,250 - ERROR: Error Controladora Device 10.117.115.181
2023-09-14 10:08:15,250 - ERROR: Error Controladora Device 10.117.115.181
2023-09-14 10:08:15,260 - ERROR: Traceback (most recent call last):
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/controladora.py", line 38, in get_data_controladora
    output = net_connect.send_command("sh wireless client mac-address " + data_pala.split()[0] + " detail")
IndexError: list index out of range

2023-09-14 10:08:15,260 - ERROR: Traceback (most recent call last):
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/controladora.py", line 38, in get_data_controladora
    output = net_connect.send_command("sh wireless client mac-address " + data_pala.split()[0] + " detail")
IndexError: list index out of range

2023-09-14 10:08:59,168 - ERROR: Error Controladora Device 10.117.115.122
2023-09-14 10:08:59,168 - ERROR: Error Controladora Device 10.117.115.122
2023-09-14 10:08:59,168 - ERROR: Traceback (most recent call last):
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/controladora.py", line 35, in get_data_controladora
    data_pala = net_connect.send_command(f"sh wireless client summary detail ipv4 | inc {device_ip}") #! Ip Pala
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/utilities.py", line 592, in wrapper_decorator
    return func(self, *args, **kwargs)
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/base_connection.py", line 1721, in send_command
    raise ReadTimeout(msg)
netmiko.exceptions.ReadTimeout: 
Pattern not detected: 'WLC\\-9800\\-MESH\\-CD>' in output.

Things you might try to fix this:
1. Explicitly set your pattern using the expect_string argument.
2. Increase the read_timeout to a larger value.

You can also look at the Netmiko session_log or debug log for more information.



2023-09-14 10:08:59,168 - ERROR: Traceback (most recent call last):
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/controladora.py", line 35, in get_data_controladora
    data_pala = net_connect.send_command(f"sh wireless client summary detail ipv4 | inc {device_ip}") #! Ip Pala
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/utilities.py", line 592, in wrapper_decorator
    return func(self, *args, **kwargs)
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/base_connection.py", line 1721, in send_command
    raise ReadTimeout(msg)
netmiko.exceptions.ReadTimeout: 
Pattern not detected: 'WLC\\-9800\\-MESH\\-CD>' in output.

Things you might try to fix this:
1. Explicitly set your pattern using the expect_string argument.
2. Increase the read_timeout to a larger value.

You can also look at the Netmiko session_log or debug log for more information.



2023-09-14 10:09:49,006 - ERROR: Error Controladora Device 10.117.115.183
2023-09-14 10:09:49,006 - ERROR: Error Controladora Device 10.117.115.183
2023-09-14 10:09:49,006 - ERROR: Traceback (most recent call last):
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/controladora.py", line 35, in get_data_controladora
    data_pala = net_connect.send_command(f"sh wireless client summary detail ipv4 | inc {device_ip}") #! Ip Pala
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/utilities.py", line 592, in wrapper_decorator
    return func(self, *args, **kwargs)
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/base_connection.py", line 1721, in send_command
    raise ReadTimeout(msg)
netmiko.exceptions.ReadTimeout: 
Pattern not detected: 'WLC\\-9800\\-MESH\\-CD>' in output.

Things you might try to fix this:
1. Explicitly set your pattern using the expect_string argument.
2. Increase the read_timeout to a larger value.

You can also look at the Netmiko session_log or debug log for more information.



2023-09-14 10:09:49,006 - ERROR: Traceback (most recent call last):
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/controladora.py", line 35, in get_data_controladora
    data_pala = net_connect.send_command(f"sh wireless client summary detail ipv4 | inc {device_ip}") #! Ip Pala
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/utilities.py", line 592, in wrapper_decorator
    return func(self, *args, **kwargs)
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/base_connection.py", line 1721, in send_command
    raise ReadTimeout(msg)
netmiko.exceptions.ReadTimeout: 
Pattern not detected: 'WLC\\-9800\\-MESH\\-CD>' in output.

Things you might try to fix this:
1. Explicitly set your pattern using the expect_string argument.
2. Increase the read_timeout to a larger value.

You can also look at the Netmiko session_log or debug log for more information.



2023-09-14 10:10:10,289 - ERROR: Error Controladora Device 10.117.115.184
2023-09-14 10:10:10,289 - ERROR: Error Controladora Device 10.117.115.184
2023-09-14 10:10:10,290 - ERROR: Traceback (most recent call last):
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/controladora.py", line 35, in get_data_controladora
    data_pala = net_connect.send_command(f"sh wireless client summary detail ipv4 | inc {device_ip}") #! Ip Pala
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/utilities.py", line 592, in wrapper_decorator
    return func(self, *args, **kwargs)
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/base_connection.py", line 1721, in send_command
    raise ReadTimeout(msg)
netmiko.exceptions.ReadTimeout: 
Pattern not detected: 'WLC\\-9800\\-MESH\\-CD>' in output.

Things you might try to fix this:
1. Explicitly set your pattern using the expect_string argument.
2. Increase the read_timeout to a larger value.

You can also look at the Netmiko session_log or debug log for more information.



2023-09-14 10:10:10,290 - ERROR: Traceback (most recent call last):
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/controladora.py", line 35, in get_data_controladora
    data_pala = net_connect.send_command(f"sh wireless client summary detail ipv4 | inc {device_ip}") #! Ip Pala
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/utilities.py", line 592, in wrapper_decorator
    return func(self, *args, **kwargs)
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/base_connection.py", line 1721, in send_command
    raise ReadTimeout(msg)
netmiko.exceptions.ReadTimeout: 
Pattern not detected: 'WLC\\-9800\\-MESH\\-CD>' in output.

Things you might try to fix this:
1. Explicitly set your pattern using the expect_string argument.
2. Increase the read_timeout to a larger value.

You can also look at the Netmiko session_log or debug log for more information.



2023-09-14 10:15:58,237 - ERROR: Error Controladora Device 10.117.115.112
2023-09-14 10:15:58,237 - ERROR: Error Controladora Device 10.117.115.112
2023-09-14 10:15:58,238 - ERROR: Traceback (most recent call last):
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/controladora.py", line 35, in get_data_controladora
    data_pala = net_connect.send_command(f"sh wireless client summary detail ipv4 | inc {device_ip}") #! Ip Pala
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/utilities.py", line 592, in wrapper_decorator
    return func(self, *args, **kwargs)
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/base_connection.py", line 1721, in send_command
    raise ReadTimeout(msg)
netmiko.exceptions.ReadTimeout: 
Pattern not detected: 'WLC\\-9800\\-MESH\\-CD>' in output.

Things you might try to fix this:
1. Explicitly set your pattern using the expect_string argument.
2. Increase the read_timeout to a larger value.

You can also look at the Netmiko session_log or debug log for more information.



2023-09-14 10:15:58,238 - ERROR: Traceback (most recent call last):
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/controladora.py", line 35, in get_data_controladora
    data_pala = net_connect.send_command(f"sh wireless client summary detail ipv4 | inc {device_ip}") #! Ip Pala
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/utilities.py", line 592, in wrapper_decorator
    return func(self, *args, **kwargs)
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/base_connection.py", line 1721, in send_command
    raise ReadTimeout(msg)
netmiko.exceptions.ReadTimeout: 
Pattern not detected: 'WLC\\-9800\\-MESH\\-CD>' in output.

Things you might try to fix this:
1. Explicitly set your pattern using the expect_string argument.
2. Increase the read_timeout to a larger value.

You can also look at the Netmiko session_log or debug log for more information.



2023-09-14 10:16:21,329 - ERROR: Error Controladora Device 10.117.115.181
2023-09-14 10:16:21,329 - ERROR: Error Controladora Device 10.117.115.181
2023-09-14 10:16:21,329 - ERROR: Traceback (most recent call last):
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/controladora.py", line 35, in get_data_controladora
    data_pala = net_connect.send_command(f"sh wireless client summary detail ipv4 | inc {device_ip}") #! Ip Pala
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/utilities.py", line 592, in wrapper_decorator
    return func(self, *args, **kwargs)
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/base_connection.py", line 1721, in send_command
    raise ReadTimeout(msg)
netmiko.exceptions.ReadTimeout: 
Pattern not detected: 'WLC\\-9800\\-MESH\\-CD>' in output.

Things you might try to fix this:
1. Explicitly set your pattern using the expect_string argument.
2. Increase the read_timeout to a larger value.

You can also look at the Netmiko session_log or debug log for more information.



2023-09-14 10:16:21,329 - ERROR: Traceback (most recent call last):
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/controladora.py", line 35, in get_data_controladora
    data_pala = net_connect.send_command(f"sh wireless client summary detail ipv4 | inc {device_ip}") #! Ip Pala
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/utilities.py", line 592, in wrapper_decorator
    return func(self, *args, **kwargs)
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/base_connection.py", line 1721, in send_command
    raise ReadTimeout(msg)
netmiko.exceptions.ReadTimeout: 
Pattern not detected: 'WLC\\-9800\\-MESH\\-CD>' in output.

Things you might try to fix this:
1. Explicitly set your pattern using the expect_string argument.
2. Increase the read_timeout to a larger value.

You can also look at the Netmiko session_log or debug log for more information.



2023-09-14 10:16:48,934 - ERROR: Error Controladora Device 10.117.115.121
2023-09-14 10:16:48,934 - ERROR: Error Controladora Device 10.117.115.121
2023-09-14 10:16:48,935 - ERROR: Traceback (most recent call last):
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/controladora.py", line 35, in get_data_controladora
    data_pala = net_connect.send_command(f"sh wireless client summary detail ipv4 | inc {device_ip}") #! Ip Pala
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/utilities.py", line 592, in wrapper_decorator
    return func(self, *args, **kwargs)
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/base_connection.py", line 1721, in send_command
    raise ReadTimeout(msg)
netmiko.exceptions.ReadTimeout: 
Pattern not detected: 'WLC\\-9800\\-MESH\\-CD>' in output.

Things you might try to fix this:
1. Explicitly set your pattern using the expect_string argument.
2. Increase the read_timeout to a larger value.

You can also look at the Netmiko session_log or debug log for more information.



2023-09-14 10:16:48,935 - ERROR: Traceback (most recent call last):
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/controladora.py", line 35, in get_data_controladora
    data_pala = net_connect.send_command(f"sh wireless client summary detail ipv4 | inc {device_ip}") #! Ip Pala
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/utilities.py", line 592, in wrapper_decorator
    return func(self, *args, **kwargs)
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/base_connection.py", line 1721, in send_command
    raise ReadTimeout(msg)
netmiko.exceptions.ReadTimeout: 
Pattern not detected: 'WLC\\-9800\\-MESH\\-CD>' in output.

Things you might try to fix this:
1. Explicitly set your pattern using the expect_string argument.
2. Increase the read_timeout to a larger value.

You can also look at the Netmiko session_log or debug log for more information.



2023-09-14 10:17:13,756 - ERROR: Error Controladora Device 10.117.115.122
2023-09-14 10:17:13,756 - ERROR: Error Controladora Device 10.117.115.122
2023-09-14 10:17:13,757 - ERROR: Traceback (most recent call last):
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/controladora.py", line 35, in get_data_controladora
    data_pala = net_connect.send_command(f"sh wireless client summary detail ipv4 | inc {device_ip}") #! Ip Pala
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/utilities.py", line 592, in wrapper_decorator
    return func(self, *args, **kwargs)
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/base_connection.py", line 1721, in send_command
    raise ReadTimeout(msg)
netmiko.exceptions.ReadTimeout: 
Pattern not detected: 'WLC\\-9800\\-MESH\\-CD>' in output.

Things you might try to fix this:
1. Explicitly set your pattern using the expect_string argument.
2. Increase the read_timeout to a larger value.

You can also look at the Netmiko session_log or debug log for more information.



2023-09-14 10:17:13,757 - ERROR: Traceback (most recent call last):
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/controladora.py", line 35, in get_data_controladora
    data_pala = net_connect.send_command(f"sh wireless client summary detail ipv4 | inc {device_ip}") #! Ip Pala
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/utilities.py", line 592, in wrapper_decorator
    return func(self, *args, **kwargs)
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/base_connection.py", line 1721, in send_command
    raise ReadTimeout(msg)
netmiko.exceptions.ReadTimeout: 
Pattern not detected: 'WLC\\-9800\\-MESH\\-CD>' in output.

Things you might try to fix this:
1. Explicitly set your pattern using the expect_string argument.
2. Increase the read_timeout to a larger value.

You can also look at the Netmiko session_log or debug log for more information.



2023-09-14 10:17:37,629 - ERROR: Error Controladora Device 10.117.115.182
2023-09-14 10:17:37,629 - ERROR: Error Controladora Device 10.117.115.182
2023-09-14 10:17:37,629 - ERROR: Traceback (most recent call last):
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/controladora.py", line 35, in get_data_controladora
    data_pala = net_connect.send_command(f"sh wireless client summary detail ipv4 | inc {device_ip}") #! Ip Pala
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/utilities.py", line 592, in wrapper_decorator
    return func(self, *args, **kwargs)
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/base_connection.py", line 1721, in send_command
    raise ReadTimeout(msg)
netmiko.exceptions.ReadTimeout: 
Pattern not detected: 'WLC\\-9800\\-MESH\\-CD>' in output.

Things you might try to fix this:
1. Explicitly set your pattern using the expect_string argument.
2. Increase the read_timeout to a larger value.

You can also look at the Netmiko session_log or debug log for more information.



2023-09-14 10:17:37,629 - ERROR: Traceback (most recent call last):
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/controladora.py", line 35, in get_data_controladora
    data_pala = net_connect.send_command(f"sh wireless client summary detail ipv4 | inc {device_ip}") #! Ip Pala
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/utilities.py", line 592, in wrapper_decorator
    return func(self, *args, **kwargs)
  File "/home/donkami/Sona/SistemaDCS/DCS-Candelaria/Services/Mesh/venv/lib/python3.10/site-packages/netmiko/base_connection.py", line 1721, in send_command
    raise ReadTimeout(msg)
netmiko.exceptions.ReadTimeout: 
Pattern not detected: 'WLC\\-9800\\-MESH\\-CD>' in output.

Things you might try to fix this:
1. Explicitly set your pattern using the expect_string argument.
2. Increase the read_timeout to a larger value.

You can also look at the Netmiko session_log or debug log for more information.



